web/cobrands/sass/_base.scss

```diff
@@ -258,18 +258,15 @@ a,
   font-size: 0.75em;
 }
 
-h4.static {
+h2.static, h2.static-with-rule {
```

Comment:
If they're being combined, might as well drop the h2.static? The point of them being separate was static-with-rule added a border-top etc that the static class did not have - if they're now the same thing, and static is unused, might as well delete it.